### PR TITLE
node:fs: fix arg parsing of readSync

### PR DESCRIPTION
**Pull request by nektro**
found this while locally setting up the nodejs test runner

https://nodejs.org/docs/latest/api/fs.html#fsreadsyncfd-buffer-offset-length-position

### CODE REVIEW

**Comment by Jarred-Sumner** [comment]
where are the tests?

File: test/js/node/fs/fs.test.ts [hunk 1]
```diff
@@ -1044,6 +1044,19 @@ describe("readSync", () => {
...
     closeSync(fd);
   });
+
+  it("works with offset + length passed but not position", () => {
+    const fd = openSync(import.meta.dir + "/readFileSync.txt", "r");
+    const four = new Uint8Array(4);
+    {
+      const count = readSync(fd, four, 0, 4);
+      const u32 = new Uint32Array(four.buffer)[0];
+      expect(u32).toBe(firstFourBytes);
```
Comment (Jarred-Sumner) [inline test/js/node/fs/fs.test.ts:1054]: where is `firstFourBytes` defined

File: test/js/node/fs/fs.test.ts [hunk 2]
```diff
@@ -1044,6 +1044,19 @@ describe("readSync", () => {
     }
     closeSync(fd);
   });
+
+  it("works with offset + length passed but not position", () => {
+    const fd = openSync(import.meta.dir + "/readFileSync.txt", "r");
+    const four = new Uint8Array(4);
+    {
+      const count = readSync(fd, four, 0, 4);
```
Comment (Jarred-Sumner) [inline test/js/node/fs/fs.test.ts:1052]: this tests length, but not offset. can this test both?
```suggestion
      const count = readSync(fd, four, 1, 5);
```

File: test/js/node/fs/fs.test.ts (hunk 1 above)
Comment (nektro) [reply test/js/node/fs/fs.test.ts:1054]: at the top of this describe block

**Pull request merged**